style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 4910px;
+  width: 320px;
+}
+
+header {
+  position: absolute;
+  top: 0;
+  height: 627px;
+  width: 320px;
```

Comment:
- [ ] Good job creating a header element, I would suggest you give it a width of 375 px or even better you can make it responsive by using `100vw` and percentages :+1: